packages/react/src/evm/components/MediaRenderer.tsx

```diff
@@ -25,6 +25,8 @@ export interface SharedMediaProps {
    * Show the media controls (where applicable) (default false)
    */
   controls?: HTMLVideoElement["controls"];
+
+  children?: any;
```

Comment:
why does this need to be any?

Reply:
It doesn't :) 
I replaced it with ReactNode